### DISCUSSION TITLE
Questão TI21#3

### DISCUSSION THREAD

**Discussion by paulocafee** [paulowh/TI21#9] [Q&A] [answered]
Qual é a diferença entre == e === em JavaScript?

**Comment by Arrowaaa** [comment]
A diferença entre == e === em javaScript:
É que ==  é comparação
e === é o sinal de igualdade

**Comment by Guilhlm** [comment, marked as answer]
== uma comparaçao do valor da variavel, === verifica tudo da variavel inclusive o seu tipo (int, float), tem q ter o mesmo valor e tipo

string ="2" == int="2" 
string ="2" !== int="2" 
int="2" === int="2"

**Comment by riichardgabriel** [comment]
o operador == compara por 'resultado'; Ele converte o que você quer comparar e verifica. Já o operador === compara o valor e o tipo, (quando você usa o operador === você força a comparar o tipo também) então só vai ser true se for exatamente igual (valor e tipo).
